etl/steps/data/garden/itopf/2023-05-18/oil_spills.countries.json

```diff
@@ -0,0 +1,27 @@
+{
+  "World": "World",
```

Comment:
`*.countries.json` is used primarily for country harmonization. In your case the only change is `nautical miles off -> off` so it might be easier to do it in pandas directly, something like `df['countries'] = df['countries'].str.replace("nautical miles off", "off")`